sdks/go.mod

```diff
@@ -52,6 +52,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require cloud.google.com/go/profiler v0.3.0
```

Comment:
Consider putting this in the above `require` block instead of it's own line.

Reply:
`go mod tidy` decided it was special enough for its own block, apparently. Fixed.